python/pyproject.toml

```diff
@@ -118,7 +118,7 @@ ollama = [
 ]
 onnx = [
     "onnxruntime-genai ~= 0.5; python_version < '3.13' and platform_system != 'Windows'",
-    "onnxruntime == 1.22.0; platform_system == 'Windows'"
+    "onnxruntime == 1.22.0"
```

Comment:
This was present because the windows package availability diverged from the MacOS/Linux availability. `onnxruntime-genai` pulls in `onnxruntime`, so we can remove it completely now that all platforms support 1.22.0.